### ISSUE TITLE
"\textunderscore" throws "Expected 'EOF'"-error

### ISSUE THREAD

**Issue by Syndesi** [Khan/KaTeX#1104]
Hi,

when I'm writing `\textunderscore` into the [demo editor](https://khan.github.io/KaTeX/), it gives me the following error message: `KaTeX parse error: Expected 'EOF', got '\textunderscore' at position 1: \̲t̲e̲x̲t̲u̲n̲d̲e̲r̲s̲c̲o̲r̲e̲`.

It seems as if it isn't supported anymore, but it's [still in the documentation](https://khan.github.io/KaTeX/function-support.html#symbols) (first column, 6th row).

**Comment by Syndesi** [comment]
I found even more symbols which are throwing error messages:
```
\textbraceleft
\textemdash
\textendash
\textgreater
\textless
\textquotedblleft
\textquotedblright
\textquoteleft
\textquoteright
\textunderscore
```

**Comment by kevinbarabash** [comment]
@Syndesi these should be pretty easy to add.  See symbols.js and look at `\textsterling` as an example.  A PR for this would be welcome if you have the time.

**Comment by Syndesi** [comment]
@kevinbarabash I think I know what to change so that these symbols are working - but when I'm testing this software with npm 5.6.0, freshly installed modules & the command `npm run start` I get this result:
![screenshot-2018-1-28 katex test](https://user-images.githubusercontent.com/9903723/35477340-db0902ca-03c0-11e8-9ba3-f0e0a150909a.png)

Error Messages:

```js
Error: Cannot find module "./fonts/KaTeX_AMS-Regular.woff2"
katex.less:7:67
ReferenceError: katex is not defined
[More Informations]
main.js:70:1
[WDS] Errors while compiling. Reload prevented.
client:154
(undefined) ./node_modules/css-loader?{"minimize":false}!./node_modules/less-loader/dist/cjs.js!./static/katex.less
Module not found: Error: Can't resolve './fonts/KaTeX_AMS-Regular.ttf' in 'C:\xampp\htdocs\KaTeX\static'
 @ ./node_modules/css-loader?{"minimize":false}!./node_modules/less-loader/dist/cjs.js!./static/katex.less 7:251-291
 @ ./static/katex.less
 @ ./katex.webpack.js
 @ multi (webpack)-dev-server/client?http://localhost:7936 ./katex.webpack.js

...

(undefined) ./node_modules/css-loader?{"minimize":false}!./node_modules/less-loader/dist/cjs.js!./static/katex.less
Module not found: Error: Can't resolve './fonts/KaTeX_Typewriter-Regular.woff2' in 'C:\xampp\htdocs\KaTeX\static'
 @ ./node_modules/css-loader?{"minimize":false}!./node_modules/less-loader/dist/cjs.js!./static/katex.less 7:5384-5433
 @ ./static/katex.less
 @ ./katex.webpack.js
 @ multi (webpack)-dev-server/client?http://localhost:7936 ./katex.webpack.js
```

I'm actually not sure if this is another bug or just using it the wrong way, I would suspect the latter. But either way, without seeing the rendered math I can't test/update this software :/

**Comment by kevinbarabash** [comment]
```
git submodule sync --recursive && git submodule update --init --recursive
```
should do the trick.  See https://github.com/Khan/KaTeX/blob/master/CONTRIBUTING.md#working-with-submodules for more details.

**Comment by Syndesi** [comment]
Thank you, but after I ran these I still got these. This time they are in a seperate file (full error log, 343 lines).
[error_log.txt](https://github.com/Khan/KaTeX/files/1670644/error_log.txt)

But to get at least something done I would work on the actual fix and upload it - maybe you/github could test it :/

**Comment by kevinbarabash** [comment]
I see, the issue is Windows doesn't support symlinks.  There's a fix that's in review right now.  See https://github.com/Khan/KaTeX/pull/1090 and https://github.com/KaTeX/katex-fonts/pull/10.  I have to fix a couple of things before the fix can be merged.

**Comment by kevinbarabash** [comment]
If you want to open a PR with your changes I can definitely test them.

**Comment by Syndesi** [comment]
Thank you, I created the pull request #1105 .
It doesn't add math-support for the `/text...`-symbols because I think `text` should indicate that this symol should be used only inside `\text{}` - therefore the error was on my side.

But I added support for €, ¥, ₹ and ₽.

**Comment by kevinbarabash** [comment]
> It doesn't add math-support for the /text...-symbols because I think text should indicate that this symol should be used only inside \text{} - therefore the error was on my side.

Right.  I should've looked at symbols.js.  It appears we already support the list of `\text...` symbols above.

**Comment by kevinbarabash** [comment]
I'm going to close this issue and track adding support for Euro, Rupee, Ruble, etc. in #1107.

**Comment by Syndesi** [comment]
Ok thank you.

**Comment by kevinbarabash** [comment]
@Syndesi I've just merged #1090 which should solve the issue you ran into earlier with the fonts being missing.
